### PR TITLE
3032 forum thread scrolling

### PR DESCRIPTION
**Pull request by trane2**
Auto scrolls to center. Works a little better, but does not exactly keep the thread at the same place.

### CODE REVIEW

File: site/app/views/forum/ForumThreadView.php [hunk 1]
```diff
@@ -569,7 +569,7 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
 						$first_post_content = html_entity_decode($first_post['content'], ENT_QUOTES | ENT_HTML5, 'UTF-8');
 
 						//replace tags from displaying in sidebar
-						$first_post_content = str_replace("[/code]", "", str_replace("[code]", "", strip_tags($first_post["content"])));
+                        $first_post_content = str_replace("[/code]", "", str_replace("[code]", "", strip_tags($first_post["content"])));
```
Comment (RylanOC) [inline site/app/views/forum/ForumThreadView.php]: Inconsistent indentation.

**Comment by bmcutler** [comment]
please resolve conflicts

**Comment by trane2** [comment]
Resolved Conflicts in new PR, see #3239

**Comment by MasterOdin** [comment]
Closed in favor of #3239

**Pull request closed without merging**